### ISSUE TITLE
Can't add new public link

### ISSUE THREAD

**Issue by kaffeeringe** [nextcloud/polls#775]
**Which Polls version are you running?**
Just updated to 1.0.0

**Nextcloud or ownCloud?:**
Nextcloud

**Nextcloud/ownCloud version:** (see Nextcloud admin page)
17.0.2.

### Steps to reproduce
1. Create a new poll
2. Click on Share
3. Click on Create Public Link
4. Error: "Fehler beim Hinzufügen der Freigabe" (Error in creating a new share)

### Expected behaviour
It fails to create a public link

### Actual behaviour
It creates a new Link

## Server configuration detail

**Operating system:** Linux 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u2 (2019-11-11) x86_64

**Webserver:** Apache/2.4.38 (Debian) (fpm-fcgi)

**Database:** mysql 10.3.18

**PHP version:** 

7.3.13-1+0~20191218.50+debian10~1.gbp23c2da
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, sodium, session, standard, cgi-fcgi, mysqlnd, PDO, xml, apcu, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, gmp, iconv, imagick, imap, intl, json, ldap, exif, memcache, mysqli, pdo_mysql, pdo_sqlite, apc, posix, pspell, readline, recode, shmop, SimpleXML, smbclient, soap, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, wddx, xmlreader, xmlrpc, xmlwriter, xsl, zip, Phar, libsmbclient, Zend OPcache

**Nextcloud version:** 17.0.2 - 17.0.2.1

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
    [core] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [stats/index.php] => Array
                        (
                            [expected] => 
                            [current] => a176a8c56ac082751dde17525236e83be3bf8ffe04ecf8e50df051bb6cc1e1790a6591562e343a3644d022fd649e1052b45128ba54fdf7bdc9c1232ae69a8e47
                        )

                )

        )

)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.3.0
 - activity: 2.10.1
 - bruteforcesettings: 1.5.0
 - calendar: 2.0.1
 - cloud_federation_api: 1.0.0
 - contacts: 3.1.8
 - dav: 1.13.0
 - encryption: 2.5.0
 - federatedfilesharing: 1.7.0
 - federation: 1.7.0
 - files: 1.12.0
 - files_downloadactivity: 1.6.0
 - files_pdfviewer: 1.6.0
 - files_rightclick: 0.15.1
 - files_sharing: 1.9.0
 - files_trackdownloads: 1.6.0
 - files_videoplayer: 1.6.0
 - firstrunwizard: 2.6.0
 - fulltextsearch: 1.3.6
 - group_everyone: 0.1.4
 - issuetemplate: 0.6.0
 - lookup_server_connector: 1.5.0
 - metadata: 0.11.1
 - news: 14.1.2
 - notifications: 2.5.0
 - oauth2: 1.5.0
 - onlyoffice: 4.1.1
 - password_policy: 1.7.0
 - polls: 1.0.0
 - privacy: 1.1.0
 - provisioning_api: 1.7.0
 - quota_warning: 1.6.0
 - recommendations: 0.5.0
 - spreed: 7.0.2
 - support: 1.0.1
 - tasks: 0.11.3
 - text: 1.1.1
 - theming: 1.8.0
 - twofactor_backupcodes: 1.6.0
 - viewer: 1.2.0
 - workflowengine: 1.7.0
Disabled:
 - admin_audit
 - comments
 - drop_account
 - files_external
 - files_trashbin
 - files_versions
 - gallery
 - logreader
 - nextcloud_announcements
 - serverinfo
 - sharebymail
 - survey_client
 - systemtags
 - updatenotification
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "vb.spd-schleswig-holstein.de",
        "spdsh.sozicloud.net"
    ],
    "memcache.local": "\\OC\\Memcache\\APCu",
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "https:\/\/vb.spd-schleswig-holstein.de\/",
    "dbtype": "mysql",
    "version": "17.0.2.1",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "appstoreenabled": true,
    "appstoreurl": "https:\/\/api.owncloud.com\/v1",
    "appstore.experimental.enabled": false,
    "apps_paths": [
        {
            "path": "\/var\/www\/clients\/client8\/web9\/web\/apps",
            "url": "\/apps",
            "writable": true
        }
    ],
    "maintenance": false,
    "theme": "",
    "htaccess.RewriteBase": "\/",
    "updatechecker": false,
    "updater.secret": "***REMOVED SENSITIVE VALUE***",
    "loglevel": 2,
    "mail_smtpmode": "smtp",
    "mail_smtpsecure": "tls",
    "mail_sendmailmode": "smtp",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "app_install_overwrite": [
        "calendar",
        "files_downloadactivity",
        "files_trackdownloads",
        "deck",
        "documents",
        "files_accesscontrol",
        "files_automatedtagging",
        "onlyoffice",
        "quota_warning",
        "registration",
        "files_retention",
        "spreed",
        "tasks",
        "groupfolders",
        "dashboard",
        "drop_account"
    ]
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** 1

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:69.0) Gecko/20100101 Firefox/69.0

**Operating system:** 

## Logs

<details><summary>Web server error log</summary>

```
Insert your web server log here 
```
</details>

<details><summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

<details><summary>Browser log</summary>

Insert your browser log here, this could for example include:

	a) The javascript console log
	b) The network log
	c) ...
</details>

**Comment by kaffeeringe** [comment]
I tested this on another instance of Nextcloud. Same versions, different server. And it works there.

**Server configuration detail**

Operating system: Linux 4.4.0-171-generic #200-Ubuntu SMP Tue Dec 3 11:04:55 UTC 2019 x86_64

Webserver: Apache (fpm-fcgi)

Database: mysql 5.6.44

PHP version: 7.3.11-nmm1

Modules loaded: Core, date, libxml, openssl, pcre, zlib, bcmath, bz2, calendar, ctype, dom, hash, filter, ftp, gettext, SPL, iconv, json, mbstring, posix, Reflection, session, standard, shmop, SimpleXML, soap, exif, sysvsem, sysvshm, tokenizer, wddx, xml, xmlreader, xmlwriter, zip, cgi-fcgi, propro, raphf, http, mysqlnd, PDO, curl, dba, fileinfo, gd, gender, gmp, htscanner, imagick, imap, interbase, intl, ldap, mailparse, mongodb, mysqli, OAuth, odbc, pdo_dblib, pdo_mysql, PDO_ODBC, pdo_pgsql, pdo_sqlite, pgsql, Phar, pspell, recode, sodium, sqlite3, tidy, xsl, ionCube Loader, Zend OPcache

Nextcloud version: 17.0.2 - 17.0.2.1

Updated from an older Nextcloud/ownCloud or fresh install:

Where did you install Nextcloud from: unknown

<details><summary>Signing status</summary>

Array ( [core] => Array ( [EXTRA_FILE] => Array ( [cgi-cron.php] => Array ( [expected] => [current] => 8257068e0c5a5d1891e81eb8b8ee820fe4d7b7d37cc44b94b5a1290b9be6757eb32c848e6a41e39796e72cd50bbe5dd0beed0158c8b6e1d14194c762fccd2bb6 )

            )

    )


)

</details>

<details><summary>List of activated apps</summary>

Enabled:
 - accessibility: 1.3.0
 - activity: 2.10.1
 - bookmarks: 2.3.4
 - calendar: 2.0.1
 - cloud_federation_api: 1.0.0
 - contacts: 3.1.8
 - dav: 1.13.0
 - encryption: 2.5.0
 - federatedfilesharing: 1.7.0
 - federation: 1.7.0
 - files: 1.12.0
 - files_external: 1.8.0
 - files_frommail: 0.3.0
 - files_markdown: 2.2.0
 - files_pdfviewer: 1.6.0
 - files_rightclick: 0.15.1
 - files_sharing: 1.9.0
 - files_trashbin: 1.7.0
 - files_videoplayer: 1.6.0
 - firstrunwizard: 2.6.0
 - gallery: 18.4.0
 - lookup_server_connector: 1.5.0
 - mail: 1.0.0
 - news: 14.1.2
 - notes: 3.1.1
 - notifications: 2.5.0
 - oauth2: 1.5.0
 - ocr: 5.0.3
 - password_policy: 1.7.0
 - polls: 1.0.0
 - privacy: 1.1.0
 - provisioning_api: 1.7.0
 - support: 1.0.1
 - survey_client: 1.5.0
 - suspicious_login: 3.0.1
 - text: 1.1.1
 - theming: 1.8.0
 - twofactor_backupcodes: 1.6.0
 - twofactor_totp: 4.1.2
 - updatenotification: 1.7.0
 - viewer: 1.2.0
 - workflowengine: 1.7.0
Disabled:
 - admin_audit
 - comments
 - files_versions
 - logreader
 - nextcloud_announcements
 - previewgenerator
 - recommendations
 - serverinfo
 - sharebymail
 - systemtags
 - user_ldap



</details>

<details><summary>Configuration (config/config.php)</summary>

{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***",
        "***REMOVED SENSITIVE VALUE***"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "17.0.2.1",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "installed": true,
    "forcessl": true,
    "theme": "",
    "maintenance": false,
    "log_type": "file",
    "logfile": "",
    "loglevel": "4",
    "logdateformat": "F d, Y H:i:s",
    "trashbin_retention_obligation": "auto",
    "mail_smtpmode": "smtp",
    "mail_smtpsecure": "tls",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "updater.release.channel": "stable",
    "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "app_install_overwrite": [
        "mail",
        "bookmarks_fulltextsearch",
        "keeweb",
        "dashboard"
    ],
    "mysql.utf8mb4": false
}


</details>

External storages: yes

<details><summary>External storage configuration</summary>

No mounts configured


</details>

Encryption: yes

User-backends:

OC\User\Database

Browser: Firefox 0.0

**Comment by dartcafe** [comment]
All my instances are working well. My theory is different behavior of the db systems. Anyway it should be solved in version 1.1.5. Currently available as RC here: https://github.com/nextcloud/polls/releases

**Comment by dartcafe** [comment]
see #776
